Commands.adoc

```diff
@@ -0,0 +1,244 @@
+export DOCKER_HOST=tcp://172.28.128.3:2376
+export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+export DOCKER_TLS_VERIFY=1
+export DOCKER_API_VERSION=1.21
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env docker)"
+----------------------------------------------------------------------------
+
+*Note:* In the case of the `docker` service, the required TLS certificates for
+securing the Docker communication will be regenerated and copied to the host
+as part of the command execution. If existing certificates are found to be
+invalid, they get regenerated.
+
+Similarly, for OpenShift you can use: +
+`$ vagrant service-manager env openshift`
+
+If you want to run the plugin and get the environment variables and certificates
+ for docker in key=value format, use:
+----------------------------------------------------------------------
+$ vagrant service-manager env docker --script-readable
+DOCKER_HOST=tcp://172.28.128.3:2376
+DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker'
+DOCKER_TLS_VERIFY=1
+DOCKER_API_VERSION=1.21
+----------------------------------------------------------------------
+
+
+== Identify IP Address and Version of the VM
+
+If you need to find information related to the VM, such as the routable
+IP address or the version and release information of the VM image, use:
+
+`$ vagrant service-manager box` +
+
+Usage: `vagrant service-manager box [sub-command] [options]`
+
+The following sub-commands can be used: +
+
+`ip`:: Displays the routable IP address of the VM +
+`version`:: Displays the version and release information of the VM +
+
+The following options can be used: +
+
+`--script-readable`:: Displays information in a key=value format +
+`-h, --help`:: Prints help for the specific command being run.
+
+=== Examples
+
+To check the routable IP address of the VM:
+---------------------------------
+$ vagrant service-manager box ip
+172.28.128.3
+---------------------------------
+
+To check the routable IP address of the VM in key=value format:
+---------------------------------------------------
+$ vagrant service-manager box ip --script-readable
+IP=172.28.128.3
+---------------------------------------------------
+
+To check the version of the VM:
+--------------------------------------
+$ vagrant service-manager box version
+Atomic Developer Bundle (ADB) 2.2.0
+--------------------------------------
+
+To check the version of the VM in key=value format:
+--------------------------------------------------------
+$ vagrant service-manager box version --script-readable
+VARIANT="Atomic Developer Bundle (ADB)"
+VARIANT_ID="adb"
+VARIANT_VERSION="2.2.0"
+--------------------------------------------------------
+
+
+== Manage Service Lifecycles
+
+If you need to mange the lifecycle of a service - check the current status of
+services, start or stop or restart a specific service use: +
```

Comment:
If you need to check the status, start, stop or restart a specific service, use: